source/ConfigSetup.h

```diff
@@ -76,7 +74,12 @@ EMP_BUILD_CONFIG(SymConfigBase,
 
     GROUP(DTH, "Settings for the Dirty Transmission Hypothesis"),
     VALUE(EFFICIENT_SYM, bool, 0, "Do you want symbionts that also have an efficiency value that evolves"),
+    VALUE(COMPETITION_MODE, bool, 0, "Should a competition between two types be conducted? (Which is specified in the code)"),
 
-
+    GROUP(SGP, "Complex Genomes Settings"),
+    VALUE(CYCLES_PER_UPDATE, size_t, 4, "Number of CPU cycles that organisms run every update"),
+    VALUE(THREAD_COUNT, size_t, 12, "Number of threads used to process organisms in parallel"),
```

Comment:
You should pull from upstream and go with the upstream default value of 1 for Threat_count

Reply:
This needs to be changed back to 1 still